code-climate-codenarc/CanNotModifyReferenceRule.groovy

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2017-Present, Niteo Consulting Private Limited, India.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or
+ * without modification, are not permitted.
+ */
```

Comment:
Why is this here? This should not be present in the open source plugin. Remove it's all occurrences.